ncm-systemd/src/main/pan/components/systemd/schema.pan

```diff
@@ -198,8 +198,10 @@ http://www.freedesktop.org/software/systemd/man/systemd.service.html
 type ${project.artifactId}_unitfile_config_service = {
     include ${project.artifactId}_unitfile_config_systemd_exec
     include ${project.artifactId}_unitfile_config_systemd_kill
+    'AmbientCapabilities' ? string[]
```

Comment:
Can we do better than just string?

A quick check seems to indicate that this should be an uppercase string that starts with `CAP_`:
```
$ grep define /usr/include/linux/capability.h | sort
#define CAP_AUDIT_CONTROL    30
#define CAP_AUDIT_READ		37
#define CAP_AUDIT_WRITE      29
…
```

See also http://man7.org/linux/man-pages/man7/capabilities.7.html.